MAUI/Autocomplete/UI-Customization.md

```diff
@@ -673,6 +715,74 @@ The following image illustrates the result of the above code:
 
 N> The `Completed` event is not supported in the Android platform.
 
+## DropDownOpening Event
+
+The `DropDownOpening` event will be fired whenever opening the dropdown menu in the SfAutocomplete. It can cancel dropdown opening with CancelEventArgs that contains the following property:
```

Comment:
The `DropDownOpening` event will be fired whenever the dropdown menu is opened in the SfAutocomplete. It can cancel dropdown opening with CancelEventArgs that contains the following property: